### ISSUE TITLE
[Claim Details V2] Complete accessibility testing for staging review

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#78461]
## Value Statement
**_As a_** Veteran
**_I want to_** use Claim Details V2 without accessibility problems
**_So that_** I can understand my claims status easily

---
### Accessibility testing on Claim Details V2

- We need to complete accessibility testing so we can prepare the accessibility artifact for the staging review.
- [Staging Artifacts for Claim Details V2 (Test Rails QA and Regression Testing)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76182)

## Designs
- [Figma link](https://www.figma.com/file/F8U4wddaFouUPVd4mGBMDI/CST---Evidence-Submission---Spec?type=design&node-id=103%3A1335&mode=design&t=Dd2iZwpTNXkF3Hzw-1)

## Tasks
- [x] Complete Axe scans
- [x] Complete keyboard testing
- [x] Complete color contrast testing
- [x] Complete Zoom testing
- [x] Complete screen reader testing

## Definition of Done
- [x] Meets acceptance criteria

## Acceptance Criteria
- [x] All test sections are complete
- [x] Accessibility artifact is filed for the epic

**Comment by davidakennedy** [comment]
I've completed the testing, but still need to create the staging accessibility artifact. Here's what I found:

One accessibility defect 1:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/78904: Enhance show past updates button so its state is understood by assistive technology - The most important one to fix in terms of experience. Should be a quick fix, adding some ARIA attributes.

Two accessibility defect 2s:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/78884:  Link text for screen reader needs to be closer to visual text - Simple text change, but may want to bring in content for a second opinion.
https://github.com/department-of-veterans-affairs/va.gov-team/issues/78864: Make Claims Process Stepper not rely on color alone - Probably the thorniest one from a visual design standpoint. cc: @skylerschain and @uxgary. We should talk through this one because there are a few options. This also may be something we could kick up to the design system because I believe this comes from a similar component elsewhere, right?

One accessibility defect 3:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/78897: Make link look like a link - This is a simple component change, and we still should be able to maintain the visual hierarchy.

Two accessibility defect 4s:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/78891: aria-describedby message not on page; remove it - Unneeded use of ARIA here, based on the page. Could be something I'm missing though.
https://github.com/department-of-veterans-affairs/va.gov-team/issues/78853: Remove disabled button on Ask for your claim decision page - Potentially more engineering work here, but worth the change to make things consistent with other parts of the experience.

Next step is to fill out the staging accessibility artifact. I'll cite these there too.

**Comment by davidakennedy** [comment]
[Accessibility artifact for the staging review is complete](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78989). I can update this when we address issues.

**Comment by skylerschain** [comment]
Thank you @davidakennedy! 

Just a couple comments on the above, documenting here for visibility and shared context: 

- #78904  and #78864 will be fixed once we roll out [Claim Contextualization](https://www.figma.com/file/bWELjWsVqkf3BDx0PXqqam/Claim-Contextualization?type=design&node-id=1-12453&mode=design&t=kmSF8FsiisBpNSVl-0). We're replacing that whole stepper with a new accordion pattern which should resolve these issues. I wonder if we can just wait until we ship Claim Contextualization to fix these. 
- #78884, #78897, and #78853 can be fixed by this ticket, which is currently in our backlog and on the cusp of prioritization. So this helps push that into sprint goals going forward.
